src/Core/Form/IdentifiableObject/DataHandler/EmployeeFormDataHandler.php

```diff
@@ -94,7 +112,10 @@ public function __construct(
         EmployeeFormAccessCheckerInterface $employeeFormAccessChecker,
         EmployeeDataProviderInterface $employeeDataProvider,
         Hashing $hashing,
-        ImageUploaderInterface $imageUploader = null
+        ImageUploaderInterface $imageUploader = null,
+        int $minLength,
+        int $maxLength,
+        int $minScore
```

Comment:
Could you move required parameters before optional ?

Reply:
I remove the null option as it's already a BC break